GermanFilter/sections/specific.txt

```diff
@@ -79,6 +79,7 @@ wiesbadenaktuell.de,rauchmeldungen.de,simfans.de,wn.de,cosmopolitan.de,wallstree
 ! ##.id-Container--proBEEP
 hallo-muenchen.de,az-online.de,lokalo24.de,rosenheim24.de,kreiszeitung.de,nordbuzz.de,mannheim24.de,merkur.de,oktoberfest-live.de,tz.de##.id-Container--proBEEP
 !
+tvmovie.de###dikr-responsive-ads-pb
```

Comment:
Element has style `display: block !important;` and overwrites usually our rule (modifying style to position -3000 should work though).
Though I do not see that left-over at all.

Reply:
@BlazDT Use AdGuard Apps.

Reply:
This rule is enough on my end.

Reply:
Hmm, after reopening site in incognito mode the !important part of style is gone and rule works. Strange.